ft232/__init__.py

```diff
@@ -18,13 +18,11 @@
 try:
     from .d2xx import list_devices, D2XXException
     from .d2xx import D2xx as Ft232
-
-    Ft232Exception = d2xx.D2XXException
+    from .d2xx import D2XXException as Ft232Exception
```

Comment:
This is a nice clean up, but can you please submit it as a separate PR? It's unrelated to this PR.